gateways/checkoutdotcom/request_builder.go

```diff
@@ -0,0 +1,59 @@
+package checkoutdotcom
+
+import (
+	"github.com/BoltApp/sleet"
+	"github.com/checkout/checkout-sdk-go/common"
+	"github.com/checkout/checkout-sdk-go/payments"
+)
+
+func buildChargeParams(authRequest *sleet.AuthorizationRequest) *payments.Request {
+	var source = payments.CardSource{
+		Type: "card",
+		Number: authRequest.CreditCard.Number,
+		ExpiryMonth: uint64(authRequest.CreditCard.ExpirationMonth),
+		ExpiryYear: uint64(authRequest.CreditCard.ExpirationYear),
+		Name: authRequest.CreditCard.FirstName + " " + authRequest.CreditCard.LastName,
+		CVV: authRequest.CreditCard.CVV,
+		BillingAddress: &common.Address{
+			AddressLine1: *authRequest.BillingAddress.StreetAddress1,
+			AddressLine2: *authRequest.BillingAddress.StreetAddress2,
+			City:         *authRequest.BillingAddress.Locality,
+			State:        *authRequest.BillingAddress.RegionCode,
+			ZIP:          *authRequest.BillingAddress.PostalCode,
+			Country:      *authRequest.BillingAddress.CountryCode,
+		},
+	}
+
+	return &payments.Request{
+		Source:   source,
+		Amount:   uint64(authRequest.Amount.Amount),
```

Comment:
how about floats?

Reply:
@pranamya-nayak amount should be coming as an integer representing cents so uint64 should be safe